dotnet-statsig/src/Statsig/Server/Evaluation/Evaluator.cs

```diff
@@ -169,9 +175,26 @@ internal ConfigEvaluation GetLayer(StatsigUser user, string layerName)
 
         private bool IsExperimentActive(ConfigSpec spec)
         {
+            bool layerAssignmentFound = false;
             foreach (var rule in spec.Rules)
             {
+                if (rule.Name == "abandoned" || rule.Name == "prestart")
+                {
+                    return false;
+                }
                 if (rule.ID.ToLowerInvariant() == "layerassignment")
+                {
+                    layerAssignmentFound = true;
+                }
```

Comment:
I think it's worth passing a flag that indicates whether the experiment is active?